src/Streamly/Internal/System/Command.hs

```diff
@@ -75,7 +76,12 @@ quotedWord =
                 '"' -> Just x
                 '\'' -> Just x
                 _ -> Nothing
-        trEsc q x = if q == x then Just x else Nothing
+        trEsc q x
+            | q == '\'' && x == '\'' = Just '\''
```

Comment:
This should return `Nothing`. 'hello\\' should return `hello\`. You can add a test for that. This should go in the `otherwise` case.

Reply:
I'll keep the case to make it explicit.

Reply:
This is `'hello\''`

Reply:
Ah, single quotes are super dumb.